include/deal.II/lac/la_vector.h

```diff
@@ -110,81 +110,88 @@ namespace LinearAlgebra
     /**
      * Multiply the entire vector by a fixed factor.
      */
-    virtual VectorSpaceVector<Number> &operator*= (const Number factor);
+    virtual Vector<Number> &operator*= (const Number factor) override;
```

Comment:
Ah, I see. I couldn't see any guards around the code or other conditional that excludes la_vector.cc from compilation without C++11. Am I just missing something or needs this be fixed before committing?